modules/shortcodes/slideshare.php

```diff
@@ -111,5 +111,10 @@ function slideshare_shortcode( $atts ) {
 
 	$player .= ' allowfullscreen webkitallowfullscreen mozallowfullscreen></iframe>';
 
-	return $player;
+	/**
+	 * Filter the returned shortcode.
+	 *
+	 * @param string $player The iframe to return.
+	 */
+	return apply_filters( 'slideshare_shortcode', $player );
```

Comment:
It might also be useful to prefix the filter with `jetpack_` to avoid conflicts with other Slideshare plugins. But I guess this is a bit outside of the scope of this PR, since the file uses non-prefixed functions already.

I could also see an additional `$attr` parameter, since it may be useful for 3rd parties trying to customize the player to know what options were provided by the author in the first place, without having to parse the final player string.